### PR TITLE
rating-subscription-get-active call added

### PR DESCRIPTION
**Pull request by akhrol**

### CODE REVIEW

File: docs/calls/rating_subscription_get_active.rst [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        then token is used for authentication instead of a password.
+
+Response
+--------
+
+Fields
+~~~~~~
+
+subscription
+    This field contains the active user subscription (object).
```
Comment (schemar) [inline docs/calls/rating_subscription_get_active.rst]: can there be only one subscription at a time? What about past (finished) subscriptions?

Comment (akhrol) [reply docs/calls/rating_subscription_get_active.rst]: Yes, only one subscription at a time.

Comment (damirius) [reply docs/calls/rating_subscription_get_active.rst]: One *active* subscription

**Comment by akhrol** [comment]
This call is only for current active subscription.

**Comment by schemar** [comment]
is there another call to get all subscriptions of the user?

**Comment by schemar** [comment]
and why separate it?

**Comment by schemar** [comment]
is that on purpose?

**Comment by akhrol** [comment]
it is what we have in the specification

**Comment by schemar** [comment]
then it is also what Julian wants 🤷‍♂️

**Comment by schemar** [comment]
Can you remave "rating" from the name (probably)? See also #88

**Comment by akhrol** [comment]
do we need to add "user-" prefix here? to be in line with other OIOI calls? 
renaming will probably lead to conflicts with current subscription calls.

**Comment by schemar** [comment]
personally I think this should replace the existing calls (in OIOI version 4)

**Pull request merged**